factory/__init__.py

```diff
@@ -266,3 +266,33 @@ def preview():
         'filetype': 'image/png',
         'resource_uri': 'pi/v1/apps/preview/%d' % counter
     }
+
+
+def extension(**kw):
+    global counter
+    counter += 1
+    slug = kw.get('slug', 'extension-%d' % counter)
+    status = 'public'
+
+    if slug == 'incomplete' or slug == 'pending':
+        status = slug
+
+    version = {
+        'id': random.randint(1, 3),
+        'download_url': URLS['extension_download'],
+        'unsigned_download_url': URLS['extension_unsigned_download'],
+        'status': status,
+        'version': '0.%d' % random.randint(1, 3)
+    }
+
+    return {
+        'id': counter,
+        'description': {'en-US': escape(kw.get('description',
```

Comment:
If `lang` get parameter is passed it should be a string. Not sure how flue handles that typically.